domain-xample/svc-xample-j/src/integrationTest/java/gov/va/vro/services/xample/XampleJavaMicroserviceTest.java

```diff
@@ -32,14 +34,22 @@ public class XampleJavaMicroserviceTest {
 
   @Autowired private RabbitAdmin rabbitAdmin;
 
-  @BeforeEach
-  private void setUp() {
-    rabbitAdmin.purgeQueue(queueName, true);
+  private final RetryTemplate retryTemplate = createRetryTemplate();
+
+  private static RetryTemplate createRetryTemplate() {
+    ExponentialBackOffPolicy backOffPolicy = new ExponentialRandomBackOffPolicy();
+    backOffPolicy.setInitialInterval(2000);
+
+    RetryTemplate retryTemplate = RetryTemplate.defaultInstance();
+    retryTemplate.setBackOffPolicy(backOffPolicy);
+    retryTemplate.setRetryPolicy(new MaxAttemptsRetryPolicy(5));
+    return retryTemplate;
   }
 
-  @AfterEach
-  private void tearDown() {
-    rabbitAdmin.purgeQueue(queueName, true);
```

Comment:
i removed this method as I thought it redundant to the `purgeQueue()` performed in `setUp()`